services/auth/test/gcp_test.js

```diff
@@ -0,0 +1,19 @@
+const assert = require('assert');
+const helper = require('./helper');
+const testing = require('taskcluster-lib-testing');
+
+helper.secrets.mockSuite(testing.suiteName(), ['gcp'], function(mock, skipping) {
+  if (mock) {
+    return;
+  }
+
+  helper.withPulse('mock', skipping);
+  helper.withEntities('mock', skipping);
+  helper.withRoles('mock', skipping);
+  helper.withServers(mock, skipping);
+  helper.withCfg(mock, skipping);
+
+  test('gcpCredentials invalid account', async () => {
```

Comment:
maybe some more test cases? 🤔

Reply:
It is still a working in progress, I am trying to figure out how to load the GCP credentials from taskcluster-secrets for testing.

Reply:
Oh, didn't know we had a label for that, thanks!!